config/organization/clusterworkspacetype-use-clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:authenticated:universal-clusterworkspacetype-use
```

Comment:
@s-urbaniak is this the naming pattern we want to use? Does it match that of kube?

Alternatively, we could use a `system:kcp` prefix.

Reply:
Renamed.

Reply:
we used the `system:kcp` prefix 👍